libraries/support/azure/service.rb

```diff
@@ -69,21 +69,25 @@ def set_reader(name, value, override)
       self
     end
 
-    def get(url:, api_version:, error_handler: nil, unwrap: nil)
+    def get(use_cache = true, url:, api_version:, error_handler: nil, unwrap: nil)
       confirm_configured!
 
-      cache.fetch(url) do
+      if use_cache
+        body = cache.fetch(url)
+      end
```

Comment:
This could be turned into a conditional clause. `body = cache.fetch(url) if use_cache`